app/templates/account/manage.html

```diff
@@ -26,8 +26,8 @@
         <tr><td>Email address</td><td>{{ user.email }}</td></tr>
         <tr><td>Phone number</td><td>{{ user.phone_number }}</td></tr>
         <tr><td>Account type</td><td>{{ user.role.name }}</td></tr>
-        {% if user.agencies|length > 0 %}
-            <tr><td>Agency affiliations</td><td>{{ user.agencies|join(',', attribute='name') }}</td></tr>
+        {% if user.is_worker() %}
+            <tr><td>Agency affiliations</td><td>{{ user.agencies|join(', ', attribute='name') }}</td></tr>
         {% endif %}
```

Comment:
Can you explain the change here? Mainly, not sure what the `|` does in the old code.